packages/web-app-vercel/fix_eslint_ver.py

```
@@ -0,0 +1,11 @@
+import json
+with open('package.json', 'r') as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)